src/AirtableClient.php

```diff
@@ -7,20 +7,19 @@
 use GuzzleHttp\Client;
 
 /**
- * Class AirtableClient
- * @package Beachcasts\Airtable
+ * Class AirtableClient.
  */
 class AirtableClient
 {
     /**
-     * Base identifier
+     * Base identifier.
```

Comment:
Please remove the edits for only adding periods. It is code, and comments shouldn't be updated with periods for the sake of it. Especially when it is not complete written sentences.

I can accept this time, if you don't want to correct, but please refrain in the future.

If you decide to follow this, please correct all places in this PR. Thanks.